tutorial/server/axum/src/main.rs

```diff
@@ -26,39 +20,15 @@ extern crate tracing;
 mod auth;
 mod startup;
 
+#[cfg(all(feature = "javascript", feature = "wasm"))]
```

Comment:
What happens when neither feature is selected?

Reply:
In that case, nothing would be served when navigating to **localhost:8080/** - that's why I marked the `wasm` feature as default